specs/phase1/shard-transition.md

```diff
@@ -44,17 +44,33 @@ def compute_shard_transition_digest(beacon_state: BeaconState,
 
 ```python
 def verify_shard_block_message(beacon_state: BeaconState,
-                               shard_state: ShardState,
+                               shard_parent_state: ShardState,
                                block: ShardBlock,
                                slot: Slot,
-                               shard: Shard) -> bool:
-    assert block.shard_parent_root == shard_state.latest_block_root
-    assert block.slot == slot
-    next_slot = Slot(beacon_state.slot + 1)
+                               shard: Shard,
+                               beacon_parent_slot: Slot=None) -> bool:
```

Comment:
What is `beacon_parent_slot` for? Isn't `beacon_state` supposed to be the beacon parent state and thus has the `beacon_parent_slot` info embedded in it?